Survey/Sources/Data/NetworkAPI/Models/APIQuestionSubmission.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  APIQuestionSubmission.swift
+//  Survey
+//
+//  Created by Khanh on 24/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+struct APIQuestionSubmission: Codable {
+
+    let id: String
+    let answers: [APIAnswerSubmission]
+
+    init(questionSubmission: QuestionSubmission) {
+        id = questionSubmission.id
+        answers = questionSubmission.answers.map(APIAnswerSubmission.init)
+    }
+}
+
+extension APIQuestionSubmission {
+
+    var asParameters: [String: Any]? {
```

Comment:
I suggest we add this as Codable extension so we can use it in every classes. Or Create a new protocol with Codable ‘protocol Parameterable: Codable’

Reply:
Good point 👍, I fixed it at https://github.com/nkhanh44/ic-survey-ios/pull/84/commits/e2c7dad35fea9d58c8f330e76d76e636cd74ad5e.